components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  enabled: boolean;
+}
+
+export default function Button({ children, enabled }: Props) {
+  if (enabled) {
+    return (
+      <button className="bg-ocf-yellow dark:bg-ocf-yellow shadow h-full w-full text-center rounded-md font-bold text-xl uppercase p-4">
+        {children}
+      </button>
+    );
+  } else {
+    return (
+      <button
+        disabled
+        className="bg-ocf-gray dark:bg-ocf-gray shadow h-full w-full text-center rounded-md font-bold text-xl uppercase"
+      >
+        {children}
+      </button>
+    );
+  }
```

Comment:
This if can be translated into `disabled={!enabled}`. When you think about it, maybe `disabled={disabled}` could  be simpler? What do you think?
```suggestion
    return (
      <button
        disabled={!enabled}
        className="bg-ocf-gray dark:bg-ocf-gray shadow h-full w-full text-center rounded-md font-bold text-xl uppercase"
      >
        {children}
      </button>
    );
```